data-es2016plus.js

```diff
@@ -3643,6 +3643,7 @@ exports.tests = [
       typescript1corejs2: typescript.fallthrough,
       typescript3_2corejs3: typescript.corejs,
       firefox68: firefox.nightly,
+      firefox71: true,
```

Comment:
Wait, I'm not sure this is true

Reply:
Yes, it is, but should be a separate PR.  My mistake, I based this one on the other instead of the master.  I'll revert it.